upgrade-notes/backend_20241119_074634.md

```diff
@@ -0,0 +1,6 @@
+#### Removing administrator password settings from administration ([#3606](https://github.com/shopsys/shopsys/pull/3606))
+
+- [features moved](#movement-of-features-from-project-base-to-packages) from project-base to the framework package:
+    - AdministratorFormType
```

Comment:
I would add
 
- method `AdministratorFacade::changePassword()` was removed, use `AdministratorPasswordFacade::setPassword()` instead
- field `Administrator::$password` is now nullable, check your usage

maybe mention that for setting the admin password we now have a new class AdminPasswordFacade